src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -12,9 +12,15 @@
 public class TicTacToeGame {
 
     private Board board;
+    private int ties;
+    private int playerXWins;
+    private int playerOWins;
 
     public TicTacToeGame(){
         board = new Board();
+        ties = 0;
+        playerXWins = 0;
```

Comment:
Name should be more meaningful (desired). 
```suggestion
        NumOfPlayerXWins = 0;
```

Reply:
Reolved by bc99163 and a7fcb0f